applications/CoSimulationApplication/custom_data_structure/pyKratos/TriangleElement.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import print_function, absolute_import, division 
```

Comment:
I just wonder if you wouldn't prefer to have the interface of this closer to the Kratos Geometry API.

Take this as myself thinking loud, not as a review